chart/values.yaml

```diff
@@ -93,7 +103,14 @@ initContainers: {}
 #     volumeMounts:
 #     - name: template
 #       mountPath: /etc/varnish/tmpl
-#
+
+extraContainers: []
+# - name: prometheus-exporter
+#   image: jonnenauha/prometheus_varnish_exporter
+#   imagePullPolicy: IfNotPresent
+#   command:
+#   - prometheus_varnish_exporter
```

Comment:
I think, even after #71 there's still a use case for adding extra containers (varnishncsa logging, for example), but #71 kind of makes this specific example obsolete.

Reply:
👍🏼 I'll leave a generic example